tests/test_sharing_client.py

```diff
@@ -0,0 +1,229 @@
+import unittest
+from unittest.mock import patch
+
+from test_utils import *
+from uid2_client import SharingClient, ClientType, EncryptionError, encryption
+
+
+@patch('uid2_client.sharing_client.refresh_sharing_keys')
+class TestSharingClient(unittest.TestCase):
+    _CONST_BASE_URL = 'base_url'
+    _CONST_API_KEY = 'api_key'
+
+    def setUp(self):
+        self._key_collection = create_key_collection(IdentityScope.UID2)
```

Comment:
could you please confirm that we're testing all unexpected fields such as `unexpected_key_field` found in C# tests?

Reply:
I had missed this earlier. Made the change in test_bidstream_client just now. Will do the same for sharing client. Thanks for pointing out

Reply:
done